bitnami/elasticsearch/templates/master-statefulset.yaml

```diff
@@ -82,9 +82,9 @@ spec:
         - name: ELASTICSEARCH_CLUSTER_HOSTS
           value: {{ template "elasticsearch.discovery.fullname" . }}
         - name: ELASTICSEARCH_CLUSTER_MASTER_HOSTS
-          {{- $elasticsearchMasterFullname := include "elasticsearch.master.fullname" . }}
-          {{- $replicas := int .Values.master.replicas }}
-          value: {{range $i, $e := until $replicas }}{{ $elasticsearchMasterFullname }}-{{ $e }} {{ end }}
+          value: {{ template "elasticsearch.master.fullname" . }}-0
```

Comment:
I don't understand this change. There can be several master node replicas, why do you hardcode it to the `-0` replica?

Reply:
I inspected the `elasticsearch.yml` file and found that it only list one at any given time (i guess there is a bug).

Reply:
Do you mean a bug in ES or our setup bash logic? Anyway, I don't think we should change this but fix the issue in that case.

Reply:
okay.. i'll remove this bit

Reply:
reverted this change. please take a look